sdk/src/context.rs

```diff
@@ -130,3 +122,10 @@ where
         Ok(())
     }
 }
+
+#[derive(AnchorDeserialize, AnchorSerialize)]
+pub struct LightInstructionInputs {
+    pub proof: Option<ProofRpcResult>,
+    pub accounts: Option<Vec<PackedCompressedAccountWithMerkleContext>>,
+    pub new_addresses: Option<Vec<PackedNewAddressParams>>,
```

Comment:
Since we want to derive the seed onchain for security it is probably a good design pattern to just send a `Option<Vec<PackedAddressMerkleContext>>` as instruction data.